### PR TITLE
Implemented Donate and Request dialogs.

### PR DESCRIPTION
**Pull request by baikauskaite**
Implemented buttons for Donate and Request, dialogs that open up when clicked.

* Implemented `DonateDialog.cshtml` and `RequestDialog.cshtml` Views for display in dialogs.
* Implemented dialog layout using Bootstrap grid and other classes.
* Implemented reusable styles for dialog text.
* Added images to dialogs.
* Implemented methods in `HomeController.cs` for opening dialog views.
* Implemented click event handler for buttons in `_Layout.cshmtl`
* Created resource files for labels.

### CODE REVIEW

File: Karma/Views/Home/DonateDialog.cshtml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="~/css/site.css" />
+</head>
+<body>
+    <div id="donateDialog" class="modal-body">
+        <div class="container-fluid">
+            <div class="row">
+                @*TODO: redirect to page with person requests*@
+                <div class="col-6 btn-image">
```
Comment (PauliusLu) [inline Karma/Views/Home/DonateDialog.cshtml:19]: is this built in css class? col-6 btn-image

**Review by PauliusLu — APPROVED** [review]
Everything is great.

**Comment by baikauskaite** [comment]
Implement Donate and Request dialogs.

File: Karma/Views/Home/DonateDialog.cshtml (hunk 1 above)
Comment (baikauskaite) [reply Karma/Views/Home/DonateDialog.cshtml:19]: col-6 is Bootstrap, btn-image is my custom class defined in site.css

**Pull request merged**